tests/run-make/print-check-cfg/rmake.rs

```diff
@@ -33,6 +33,21 @@ fn main() {
         /*has_any_any*/ false,
         /*contains*/ &["unix", "miri", "feature"],
     );
+    check(
+        /*args*/ &["--check-cfg=cfg(feature, values())"],
+        /*has_any*/ false,
+        /*has_any_any*/ false,
+        /*contains*/ &["feature="],
+    );
```

Comment:
Suggestion (non-blocking): we could just have `check()` here take a `CheckArgs` struct literal:

```rs
check(CheckArgs {
    args: &["--check-cfg=cfg(feature, values())"],
    has_any: false,
    has_any_any: false,
    contains: &["feature="],
})
```

Reply:
Done in https://github.com/rust-lang/rust/pull/125822 (let's continue there)